docs/courses/level-one/chapter-5/chapter-5.6.md

```diff
@@ -29,10 +31,12 @@ In the Discord node window, configure these parameters:
 - **Message**:
     - Select the **Expression** tab on the right side of the Message field.
     - Copy the text below and paste it into the **Expression** window, or construct it manually using the **Expression Editor**.
-        - `This week we've {{$json["totalBooked"]}} booked orders with a total value of {{$json["bookedSum"]}}. My Unique ID: {{ $('HTTP Request').params["headerParameters"]["parameters"][0]["value"] }}`
+		```
+		This week we've {{$json["totalBooked"]}} booked orders with a total value of {{$json["bookedSum"]}}. My Unique ID: {{ $('HTTP Request').params["headerParameters"]["parameters"][0]["value"] }}
+		```
 
         /// note | Constructing your own message
-        To add the Unique ID portion of the statement, you'll need to expand **Nodes** > **HTTP Request** > **Parameters** > **headerParameters** > **parameters** > **[Item: 0]** and select the **value**.
+        To add the Unique ID part of the statement, you'll need to expand **Nodes** > **HTTP Request** > **Parameters** > **headerParameters** > **parameters** > **[Item: 0]** and select the **value**.
```

Comment:
This does not actually seem possible to do in the current UI anymore. Is there a way to discover this information in the current iteration other than abusing tab completion?

Reply:
According to David, there might not be a good way to find this. I'm going to remove the note for now since it's out dated, but it'd be great to think of something else to do for this eventually.